nx-dev/ui-powerpack/src/lib/powerpack-features.tsx

```diff
@@ -0,0 +1,262 @@
+'use client';
+import { forwardRef, ReactElement, ReactNode, useRef } from 'react';
+import { ButtonLink, SectionHeading, Strong } from '@nx/nx-dev/ui-common';
+import { cx } from '@nx/nx-dev/ui-primitives';
+import { AnimatedAngledBeam } from '@nx/nx-dev/ui-animations';
+import {
+  CalendarDaysIcon,
+  CircleStackIcon,
+  ServerIcon,
+} from '@heroicons/react/24/outline';
+import { NxIcon } from '@nx/nx-dev/ui-icons';
+
+export function PowerpackFeatures(): ReactElement {
+  return (
+    <section className="relative isolate">
+      <div className="mx-auto max-w-7xl px-6 lg:px-8">
+        <div className="grid grid-cols-1 gap-8 lg:grid-cols-2">
+          <div className="col-span-full flex max-w-full flex-col gap-16 bg-white/50 px-6 py-16 ring-1 ring-slate-200 sm:rounded-3xl sm:p-8 lg:mx-0 lg:max-w-full lg:flex-row lg:items-center lg:py-16 xl:px-16 dark:bg-white/5 dark:ring-white/10">
+            <div className="xl:max-w-xl">
+              <SectionHeading
+                as="h2"
+                variant="title"
+                id="self-hosted-cache-storage"
+                className="scroll-mt-48"
+              >
+                Self-hosted cache storage
+              </SectionHeading>
+              <p className="mt-6 text-pretty text-lg">
+                Nx Powerpack enables you to use <Strong>AWS S3</Strong> or a{' '}
```

Comment:
```suggestion
                Nx Powerpack enables you to use <Strong>Amazon S3</Strong> or a{' '}
```

Reply:
doesn't probably matter a lot, but in the docs we also mention mostly "AWS S3" everywhere